libraries/AP_Motors/AP_MotorsCoax.h

```diff
@@ -49,6 +49,8 @@ class AP_MotorsCoax : public AP_MotorsMulticopter {
     //  this can be used to ensure other pwm outputs (i.e. for servos) do not conflict
     uint16_t            get_motor_mask() override;
 
+    const char* get_frame_string() override { return "COAX"; }
```

Comment:
```suggestion
    const char* get_frame_string() const override { return "COAX"; }
```

... that will need to be done on all of them

Reply:
done